packages/mdc-menu/README.md

```diff
@@ -237,7 +237,7 @@ Method Signature | Description
 Method Signature | Description
 --- | ---
 `handleKeydown(evt: Event) => void` | Event handler for the `keydown` events within the menu.
-`handleClick(evt: Event) => void` | Event handler for the `click` events within the menu.
+`handleItemAction(index: number) => void` | Event handler for list's action event.
 `handleSelection(listItem: Element) => void` | Handler for a selected list item. Use this instead of `handleClick` when you don't have access to list item click event.
```

Comment:
Rename `handleClick` to `handleItemAction`

Reply:
Thanks for pointing this out.

I removed redundant / unused `handleSelection` method from foundation. `handleItemAction` replaces `handleSelection`